modules/search/client/views/search-sidebar-filters.client.view.html

```diff
@@ -6,6 +6,52 @@ <h4>Hosts or meetups?</h4>
   </div>
   <!-- /Offer type filter -->
 
+  <!-- Number of People offer maxGuests filter -->
+  <fieldset class="offer-maxguests">
+    <legend>
+      <h4 id="maxGuests">How many persons can the host accommodate?</h4>
```

Comment:
I wonder if there's shorter ways to say the same? This works fine too tho.

Reply:
1. How many persons can the host accommodate ?
2. Persons host can accommodate ?
3. Number of persons host can c ?
4. Max number of guests allowed ?
5. Max guests host can accommodate ?
6. Max number of guests allowed ?